templates/media/album-gallery.php

```diff
@@ -72,7 +72,8 @@
 	<?php } else { ?>
 		<p class="rtmedia-no-media-found">
 			<?php
-			apply_filters( 'rtmedia_no_media_found_message_filter', esc_html_e( 'Sorry !! There\'s no media found for the request !!', 'buddypress-media' ) );
+			$message = apply_filters( 'rtmedia_no_media_found_message_filter', 'Sorry !! There\'s no media found for the request !!' );
+			echo esc_html__( $message, 'buddypress-media' );
```

Comment:
@JAshwini You can use `esc_html_e` instead. Please check at all instances.

Reply:
@SID177 
For following syntax `esc_html_e()` and `echo esc_html__()`, there is no difference, either of them can be used so there is no need to replace.